backend/python/app/__init__.py

```diff
@@ -47,6 +49,8 @@ def create_app(config_name):
     app.config["CORS_SUPPORTS_CREDENTIALS"] = True
     CORS(app)
 
+    Limiter(app, key_func=get_remote_address, default_limits=["15 per minute"])
```

Comment:
The rate limit seems to be on the lower side, though I'm not too sure what a reasonable value should be. How about we set a new env variable called `BACKEND_API_DEFAULT_PER_MINUTE_RATE_LIMIT` to allow users to configure this value? We can leave 15 as the default.

```suggestion
    Limiter(app, key_func=get_remote_address, default_limits=[f"{os.getenv("BACKEND_API_DEFAULT_PER_MINUTE_RATE_LIMIT") or 15} per minute"])
```

Also, how easy would it be to override the rate limit for individual endpoints? It seems that we need to be a bit more strict with Firebase operations, see Firebase rate limits: https://firebase.google.com/docs/auth/limits

Reply:
Discussed during meeting, made the default rate limit configurable. Since it is per endpoint, leaving 15 as the default.

Overriding the rate limit looks simple. For TypeScript, they can create a new rate limiter and add it as a middleware for the route similar to other middlewares we have. For GraphQL, they can define all overridden rate limits in the shield function in graphql index.ts. For Python, they can annotate routes using the limiter object created by the Limiter call